src/test/java/APICalls/BaseCall.java

```diff
@@ -0,0 +1,53 @@
+package APICalls;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import JacksonModels.Response;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.util.EntityUtils;
+import utility.PropertyManager;
+import java.io.IOException;
+
+import static io.restassured.RestAssured.given;
+
+
+public class BaseCall {
+
+    public static final String apiKey;
+
+    static {
```

Comment:
you have multiple static blocks initialising different variables
these can be consolidated into a single static block to improve readability and maintain organisation